src/assets-webkit/styles/serlo-tailwind.css

```diff
@@ -266,74 +266,32 @@
   }
 
   .serlo-editor-hacks {
-    h1 {
-      @apply mx-side mb-9 mt-4 p-0 text-3.5xl font-bold special-hyphens-auto;
+    @apply text-lg leading-cozy;
+
+    .serlo-h1 {
+      @apply special-hyphens-auto;
     }
-    h2 {
-      @apply mt-0 mb-6 pb-1 pt-6;
-      @apply text-2.5xl font-bold special-hyphens-auto;
-      @apply border-b border-gray-300 text-gray-900;
+    .serlo-h2 {
+      @apply !mx-0;
     }
-
-    h3 {
-      @apply mt-5 mb-8 pt-3 text-1.5xl font-bold text-gray-900;
+    .serlo-h3 {
+      @apply !mx-0 !mt-5;
     }
-    div[contenteditable] h3 {
+
+    div[contenteditable] .serlo-h3 {
       @apply mt-0;
     }
 
-    @apply text-lg leading-cozy;
-
     a[data-key] {
-      @apply break-words text-brand no-underline special-hyphens-auto hover:underline;
+      @apply special-hyphens-auto;
     }
 
     code {
       @apply rounded-sm bg-brand-100 p-1 text-base text-brand;
     }
 
-    /* 
-    good for some, bad for others
-    [data-slate-object='block'] {
-      @apply mb-block;
-    } */
-
-    ul:not(.unstyled-list) {
-      @apply mx-side mb-block mt-4 list-none pl-5;
-
-      & > li:before {
-        @apply absolute bg-brand-400 special-content-space;
-        @apply -ml-5 mt-2.25 h-2.5 w-2.5 rounded-full;
-      }
-      & > li {
-        @apply mb-2;
-      }
-      & > li > ul,
-      & > li > ol {
-        @apply mt-2 !mb-4;
-      }
-    }
-    ol {
-      @apply mx-side mb-block mt-0 list-none pl-7;
-      @apply special-reset-list-counter;
-
-      & > li:before {
-        @apply absolute special-content-list-counter special-increment-list-counter;
-        @apply -ml-7 rounded-full text-center font-bold;
-        @apply mt-0.5 h-4 w-4 bg-brand-200 text-xs;
-        @apply pt-0.25 leading-tight text-brand;
-      }
-      & > li {
-        @apply mb-2;
-      }
-      & > li > ul,
-      & > li > ol {
-        @apply mt-2 !mb-4;
-      }
-    }
-
     blockquote {
-      @apply my-block mx-side border-l-6 border-brand-200 p-2.5;
+      @apply serlo-blockquote;
```

Comment:
@elbotho Have you tested if this is working? I remembered that there where some limitations with applying custom classes.

You can add the class to the `blockquote-renderer` directly, though - or just delete it ...

Reply:
added it directly, will do more cleanup here after this PR and also [this](https://github.com/serlo/frontend/pull/2527) is merged.